tests/test_venue_request.py

```diff
@@ -0,0 +1,139 @@
+import openreview
+import pytest
+import time
+from selenium.common.exceptions import NoSuchElementException
+from venue_request import VenueRequest
+
+class TestVenueRequest():
+
+    def test_venue_request_setup(self, client):
+        
+        super_id = 'openreview.net'
+        support_group_id = super_id + '/Support'
+        venue = VenueRequest(client, support_group_id=support_group_id, super_user='openreview.net')
+
+        assert venue.support_group.id == support_group_id
+        assert venue.bid_stage_super_invitation
+        assert venue.decision_stage_super_invitation
+        assert venue.meta_review_stage_super_invitation
+        assert venue.review_stage_super_invitation
+        assert venue.meta_review_stage_super_invitation
+
+        assert venue.deploy_super_invitation
+        assert venue.comment_super_invitation
+        assert venue.recruitment_super_invitation
+
+    def test_venue_request_post_deploy_revise(self, client, selenium, request_page, helpers):
+        
+        super_id = 'openreview.net'
+        support_group_id = super_id + '/Support'
+        VenueRequest(client, support_group_id, super_id)
+        
+        time.sleep(5)
+        request_page(selenium, 'http://localhost:3000/group?id={}&mode=default'.format(support_group_id), client.token)
+
+        helpers.create_user('new_test_user@mail.com', 'Newtest', 'User')
+        request_form_note = client.post_note(openreview.Note(
+            invitation=support_group_id +'/-/Request_Form',
+            signatures=['~Newtest_User1'],
+            readers=[
+                support_group_id,
+                '~Newtest_User1',
+                'new_test_user@mail.com',
+                'tom@mail.com'
+            ],
+            writers=[],
+            content={
+                'title': 'Test 2021 Venue',
+                'Official Venue Name': 'Test 2021 Venue',
+                'Abbreviated Venue Name': 'TestVenue@OR2021',
+                'Official Website URL': 'https://testvenue2021.gitlab.io/venue/',
+                'program_chair_emails': [
+                    'new_test_user@mail.com',
+                    'tom@mail.com'],
+                'contact_email': 'new_test_user@mail.com',
+                'Area Chairs (Metareviewers)': 'No, our venue does not have Area Chairs',
+                'Venue Start Date': '2020/02/01',
```

Comment:
you can set these dates using the current time otherwise we can have tests failing on 2025!

Reply:
Agreed. Updated.